app/frontend/src/components/admin/StatusTable.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <v-card>
+    <v-toolbar flat color="grey lighten-3">
+      <v-card-title>Submission Status History</v-card-title>
+    </v-toolbar>
+
+    <v-container>
+      <v-alert v-if="showAlert" :type="alertType" tile dense>{{alertMessage}}</v-alert>
+
+      <v-data-table
+        :headers="headers"
+        :items="statuses"
+        :items-per-page="10"
+        :search="search"
+        :loading="loading"
+        loading-text="Loading... Please wait"
+        item-key="inspectionStatusId"
+        class="status-table"
+      >
+        <template v-slot:item.createdAt="{ item }">{{ formatDate(item.createdAt) }}</template>
+
+        <template v-slot:item.grade="{ item }">
+          <v-icon v-if="item.grade && item.grade === 'PASS'" color="green">check</v-icon>
```

Comment:
best use toUppercase for a check tp PASS.

Reply:
👍 Will add a check for that.